media-sound/snapcast/snapcast-9999.ebuild

```diff
@@ -49,7 +50,9 @@ pkg_setup() {
 src_configure() {
 	local mycmakeargs=(
 		-DBUILD_CLIENT=$(usex client)
+		-DBUILD_WITH_EXPAT=$(usex expat)
 		-DBUILD_WITH_FLAC=$(usex flac)
+		-DBUILD_WITH_OPUS=$(usex opus)
 		-DBUILD_SERVER=$(usex server)
 		-DBUILD_STATIC_LIBS=$(usex static-libs)
 		-DBUILD_TESTS=no
```

Comment:
Do tests work?

Reply:
I can't seem to find any tests, `make test` does nothing?? Weird, or I am missing something...